project/yahtzee/frontend/templates/board.html

```diff
@@ -264,7 +264,9 @@ <h1>Select Dice To Save:</h1>
                 <br>
                 <input type="hidden" id="playerId" value="{{ score.player.id }}" />
                 <input type="hidden" id="gameId" value="{{ game.game_id }}" />
-                <a href="{% url 'end' %}">End Game |</a>
+                {% if game.host == hand.player and hand.rem_rounds < 0 %}
```

Comment:
Should this be == 0 instead of less than 0?

Reply:
hand.rem_rounds is -1 once the last score is input. 

the same logic is used here:
{% if hand.rem_rounds < 0 %}
                  <h1>Game Over!</h1>
                  <h1>Final Score: {{score.sumValues}}</h1>

Reply:
Ahhh gotcha, my apologies!